client/my-sites/stats/stats-list/stats-list-item.jsx

```diff
@@ -191,7 +197,13 @@ module.exports = React.createClass( {
 				icon = ( <span className="stats-list__flag-icon" style={ style } /> );
 			}
 
-			return ( <span className={ wrapperClassSet } key={ i } >{ gridiconSpan }{ icon }<Emojify>{ labelItem.label }</Emojify></span> );
+			if ( this.props.data.link ) {
```

Comment:
Just noticed that `this.props.data` is assigned to `data` above, so this can just use the shorter version.